frontend/src/metabase/css/components/modal.css

```diff
@@ -64,57 +64,57 @@
 
 .Modal-backdrop.Modal-appear,
 .Modal-backdrop.Modal-enter {
-  transition: background-color 200ms ease-in-out;
   background-color: color-mod(var(--color-bg-black) alpha(-99%));
 }
 
 .Modal-backdrop.Modal-appear-active,
 .Modal-backdrop.Modal-enter-active {
+  transition: background-color 200ms ease-in-out;
```

Comment:
Fix modal transition not always working. [From this doc.](https://reactcommunity.org/react-transition-group/css-transition)

> `*-active` classes represent which styles you want to animate **to**, so it's important to add `transition` declaration only to them, otherwise transitions might not behave as intended!